hiredis-client/ext/redis_client/hiredis/extconf.rb

```diff
@@ -31,9 +31,10 @@
   end
 
   Dir.chdir(hiredis_dir) do
-    flags = ["static", "USE_SSL=1"]
+    flags = ["static", "USE_SSL=1", "CFLAGS='-fvisibility=hidden'"]
     if openssl_lib
-      flags << %(CFLAGS="-I#{openssl_include}") << %(SSL_LDFLAGS="-L#{openssl_lib}")
+      flags << "CFLAGS='-I#{openssl_include}'"
```

Comment:
This will likely just override the previous value of CFLAGS

Reply:
Yeah, I'm not sure how to combine both. I tried `CFLAGS="-fvisibility=hidden -I..."` and it wasn't happy.

Somehow it seems like this current way worked.

Reply:
Ah nevermind, you are right I now have the opposite issue.